dictionaries/CallMap_80_delta.php

```diff
@@ -53,10 +53,6 @@
       'old' => ['int|false'],
       'new' => ['int'],
     ],
-    'DateTime::setTime' => [
```

Comment:
These never officially returned false. The return from an invalid parameter was always undefined behavior and PHP does not document these changes when parameter parsing throws instead of returning null/false.